### PR TITLE
add recipe for mpg123

### PR DESCRIPTION
**Pull request by hbredin**

### CODE REVIEW

**Comment by hbredin** [comment]
@conda-forge/core Is merging done automatically or do we need someone to pull the trigger?

Same question goes for my other two pull requests: 
 - [argtable2](https://github.com/conda-forge/staged-recipes/pull/2688)
 - [libsndfile](https://github.com/conda-forge/staged-recipes/pull/2689)

These three packages are required for a subsequent [`Yaafe`](https://github.com/Yaafe/Yaafe) recipe.

File: recipes/mpg123/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{% set name = "mpg123" %}
+{% set version = "1.24.0" %}
+{% set md5 = "75d62ac0cb713a7bac5af4ded4af2bb4" %}
```
Comment (ericdill) [inline recipes/mpg123/meta.yaml]: Please use sha256 here.

File: recipes/mpg123/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+requirements:
+  build:
+    - gcc
+
+test:
+  commands:
+    - test -f $PREFIX/lib/libmpg123.dylib  # [osx]
+    - test -f $PREFIX/lib/libout123.dylib  # [osx]
+    - test -f $PREFIX/lib/libmpg123.so  # [linux]
+    - test -f $PREFIX/lib/libout123.so  # [linux]
```
Comment (ericdill) [inline recipes/mpg123/meta.yaml]: Please add conda inspect checks to the test/commands section:

```yaml
    - conda inspect linkages -p $PREFIX $PKG_NAME # [linux]
    - conda inspect objects -p $PREFIX $PKG_NAME  # [osx]
```

**Comment by ericdill** [comment]
Please also add `toolchain` as a build dependency. More info here: https://github.com/conda-forge/staged-recipes/pull/1674

**Comment by ericdill** [comment]
I am not an expert on compilations with conda-build, but this seems good to me. If there are any issues with the resultant package (linkages against system libraries instead of conda libraries), those can be fixed in the feedstock.

**Comment by hbredin** [comment]
Great. Thanks @ericdill !

**Comment by jakirkham** [comment]
Somehow this recipe did not get converted. Not sure why ATM, but am re-adding it for conversion with PR ( https://github.com/conda-forge/staged-recipes/pull/2722 ). Please take a look to make sure it is correct.

**Pull request merged**